lib/krikri/harvesters/marc_xml_harvester.rb

```diff
@@ -0,0 +1,106 @@
+module Krikri::Harvesters
+
+  class MarcXMLHarvester
+    include Krikri::Harvester
+
+    attr_reader :opts
+
+    def initialize(opts = {})
+      @max_records = opts.fetch(:max_records, :unlimited)
```

Comment:
Is there a use case for `:max_records` as an option? 

Typically when we want to harvest a small number of records we do `harvester.records.take(n)`, relying on the `#lazy` factor. Obviously that can't be passed in as an option to the job; if we think there's a good reason to do that, in general, perhaps we should add this to the base `Harvester` using something like the `#take(n)` implementation.